wdl/GATKSVPipelineSingleSample.wdl

```diff
@@ -364,7 +362,6 @@ workflow GATKSVPipelineSingleSample {
 
     RuntimeAttr? runtime_override_update_sr_list
     RuntimeAttr? runtime_override_merge_pesr_depth
-    RuntimeAttr? runtime_override_merge_pesr_depth
```

Comment:
@mwalker, I don't know if this is a copy-paste error, or the `breakpoint_overlap_filter` is intentionally assigned from `merge_pesr_depth`.

Reply:
This does look like an error, can you add a `runtime_override_breakpoint_overlap_filter` input?